api/krusty/testdata/helmcharts/multiple-values-files/charts/test-chart/values.yaml

```diff
@@ -0,0 +1 @@
+abc: def
```

Comment:
This doesn't seem to be used (and therefore isn't tested)

Reply:
removed